src/kafka/mod.rs

```diff
@@ -134,4 +157,48 @@ impl CommitLog for KafkaRestCommitLog {
             }
         })
     }
+
+    async fn produce(&self, record: &ProducerRecord) -> Result<ProducedOffset, ProducerError> {
+        match self
+            .client
+            .post(
+                self.server
+                    .join(&format!("/topics/{}", record.topic))
+                    .unwrap(),
```

Comment:
Can this unwrap fail? If so would it be preferable to map the error to a ProducerError and return that rather than panicking?

Reply:
If it fails then it is a logic issue.